addons/website_payment/models/payment_provider.py

```diff
@@ -31,6 +31,14 @@ def _get_compatible_providers(self, *args, website_id=None, **kwargs):
             )
         return providers
 
+    def activate_stripe(self):
+        if not self.env.company.country_id.is_stripe_supported_country:
```

Comment:
Why not `self.is_stripe_supported_country`, which is a related to `self.company_id.country_id.is_stripe_supported_country`, as was done before ?

Reply:
because this field is the field of the res.country model, there is no such field in payment.provider

Reply:
But we use it only once so far. Do we need to add a new field for that?